make_dashboards/make_dashboards.py

```diff
@@ -33,8 +33,9 @@
 @click.option('--couch_server', '-s', required=True)
 @click.option('--outdir', '-o', required=True, help = "Create dashboards in the specified output directory.")
 @click.option('--demo', is_flag=True)
+@click.option('--genstat_url', '-g', default="https://genomics-status.scilifelab.se")
```

Comment:
Nice! I assume this is for simpler testing with stage or something?

Reply:
Yeah or your locally running gen-stat. I was getting annoyed about hard-coding `localhost:1971` only to forget to change it back later :wink: